GOVERNANCE.md

```diff
@@ -0,0 +1,25 @@
+# VS Code Kubernetes Tools Project Goveranance
+
+This document outlines how the Kubernetes extension project governs itself.  This
+comprises the extension itself, the API package and the API samples.
+
+Everyone who interacts with the project must abide by our [Code of Conduct].
```

Comment:
Did you forget to add the `Code of Conduct` file?

Reply:
Oof, thanks for catching that.  Added and also changed the contributing and readme to point to the CoC file instead of having duplicate copies.